### PR TITLE
Testing features with comment

### PR DESCRIPTION
**Pull request by meronogbai**
- Added rspec files to test units
- Tested all relevant methods

### CODE REVIEW

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Meron Bsrat & Avi Karmaker** 
Nice job with your first submission :muscle:  :muscle: 
Your project meets most of the requirements :clap: :clap: :clap:  
We still need to fix some details:

- [x] rename your spec files, change "game_board" with "game_board_spec" and "player" with "player_spec".
- [x] In your test files use "require_relative" instead of "require".
- [x] Add some additional tests: test first player wins the game, test second player wins the game, test draw game.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: spec/game_board_spec.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      expect(game.taken_fields.length).to eql 9
+    end
+  end
+  describe '#field_validation' do
+    game = Game.new
+    game.taken_fields = [1, 7, 2, 6, 3]
+    it 'Checks if field taken' do
+      expect(game.field_validation(1)).to eql(0)
+      expect(game.field_validation(10)).not_to eql(0)
+    end
```
Comment (euqueme) [inline spec/game_board_spec.rb]: - [x] let's create one `it block` for each of the expected behaviors

File: spec/game_board_spec.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      expect(game.did_i_win).to eql(2)
+    end
+  end
+  describe '#who_is_next' do
+    game = Game.new
+    game.taken_fields = [1, 7, 2, 6, 3]
+    it 'Checks if turn changer works' do
+      expect(game.who_is_next).to eql(2)
+      expect(game.who_is_next).not_to eql(1)
+    end
```
Comment (euqueme) [inline spec/game_board_spec.rb]: - [x] let's create one `it block` for each of the expected behaviors

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @meronokbay and @marcode95 :lollipop: :balloon: 

### Thank you for working on the changes requested by the previous reviewer :medal_sports: :medal_sports: you have a good amount of test cases and most of them are following the right structure :ok_hand: let's work on the following changes :construction: :construction_worker: :construction_worker: 

#### Project specific

* [x] Specs for each method should be wrapped in one `describe` block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/describe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

> let's create test cases for your `update` and `display_board` methods of the Board class

* [x] You should have one `it` block for each expected behaviour. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/itblock.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

> I've put comments in your code where I found describe blocks with more than one expected behavior  

- [x] Let's create a spec file to test the Game class and another to test the Board class instead of doing it in the same file `game_board_spec.rb`

### [OPTIONAL]

- [x] Let's add instructions in your Readme file of how to properly run your test cases with RSpec

### That's it, team! :raised_hands: 

### Please :pray: do the changes :technologist: :technologist: and submit back for a code review when you are done :memo: 

Happy doing! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team  ,  you have done a brilliant job with this project. :clap:
#### You can merge !!!

**Pull request merged**